tests/certification/state/azure/cosmosdb/README.md

```diff
@@ -0,0 +1,18 @@
+# Azure Cosmos DB certification testing
+
+This project aims to test the Azure Cosmos DB State Store component under various conditions.
+
+## Test plan
+
+### Basic Test using master key authentication
+1. Able to create and test connection.
+2. Able to do set, fetch and delete.
+
+### Test Partition Keys
+Ensure the following scenarios:
+ 1. In case of invalid partition key, the access fails
```

Comment:
Access? That sounds like it's about authorization.. but that's not so.

Please be specific that this is about a Get or Set operation.